tests/models/mistral/test_modeling_mistral.py

```diff
@@ -447,17 +447,27 @@ def test_model_7b_logits(self):
         print(out[0, 0, :30])
         torch.testing.assert_close(out[0, 0, :30], EXPECTED_SLICE, atol=1e-4, rtol=1e-4)
 
+        del model
+        torch.cuda.empty_cache()
+        import gc
+        gc.collect()
+
     @slow
     def test_model_7b_generation(self):
         EXPECTED_TEXT_COMPLETION = (
-            """My favourite condiment is mayonnaise. I love it on sandwiches, in salads, on burgers"""
+            """My favourite condiment is 100% ketchup. I love it on everything. I’m not a big"""
```

Comment:
This is given by the contributor, where we don't know their environment. On T4, this is the first time we make this test run.